### PR TITLE
Cirros for hotplug tests

### LINKED ISSUE #5667: Replace fedora VMs for hotplug tests with cirros
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug

/kind enhancement

**What happened**:

The hotplug tests use fedora because the assumption was made that cirros does not support scsi hotplug.
However cirros supports hotplug:

```
[    9.729213] sd 0:0:0:0: [sda] 123863 512-byte logical blocks: (63.4 MB/60.5 MiB)
[    9.756446] sd 0:0:0:0: [sda] Write Protect is off
[    9.765072] sd 0:0:0:0: [sda] Mode Sense: 63 00 00 08
[    9.765462] sd 0:0:0:0: [sda] Write cache: enabled, read cache: enabled, doesn't support DPO or FUA
[    9.784728] sd 0:0:0:0: [sda] Attached SCSI disk
```

**What you expected to happen**:

The hotplug storage tests should be switch to cirros, since fedora is quite heavy.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Switch the VM OS to cirros instead of fedora to reduce resource usage.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #5667

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/storage/hotplug.go [hunk 1]
```diff
@@ -508,6 +508,8 @@ var _ = SIGDescribe("Hotplug", func() {
 			for _, volumeName := range dvNames {
 				By("removing volume " + volumeName + " from VM")
 				removeVolumeFunc(vm.Name, vm.Namespace, volumeName)
+			}
+			for _, volumeName := range dvNames {
```
Comment (enp0s3) [inline tests/storage/hotplug.go:512]: @awels Why did you move the `verifyVolumeNolongerAccessible` to a dedicated volume iteration?

**Review by enp0s3 — CHANGES_REQUESTED** [review]
@awels Thank you. A small question below

File: tests/storage/hotplug.go (hunk 1 above)
Comment (awels) [reply tests/storage/hotplug.go:512]: I encountered some issues where if I immediately tried to check if if the device was gone from inside the guest, it would prevent the disk from being removed. So instead of immediately checking, I decided to remove them all first and then check if they were gone. This would give enough time between removing and checking and it passes.

**Review by enp0s3 — APPROVED** [review]
@awels @rmohr Sorry for the delay. I doubt that we need to use `Eventually` in this case but I see we already use it in `verifyVolumeNolongerAccessible`
/lgtm

**Comment by awels** [comment]
So we have retried the sig compute 1.19 about a dozen times now, I think there is a problem with it. Especially since this PR doesn't touch any actual code and just updates the hotplug tests. How can we move forward with this besides just hitting /retest until hopefully it will pass?

**Comment by xpivarc** [comment]
> So we have retried the sig compute 1.19 about a dozen times now, I think there is a problem with it. Especially since this PR doesn't touch any actual code and just updates the hotplug tests. How can we move forward with this besides just hitting /retest until hopefully it will pass?

@awels 
The problematic test is being moved to quarantine, see [PR](https://github.com/kubevirt/kubevirt/pull/5790).

**Comment by awels** [comment]
Okay cool so I just keep retrying until I get lucky, or can we have someone override it?

**Comment by brybacki** [comment]
@awels I suppose someone with "admin" rights could override the 1.19 compute lane, since only the well known test failed. And as yoiu said, this change only updates storage tests, does not change any code under test. 

The alternative is to rebase and retest everything.

**Comment by fgimenez** [comment]
/retest

The PR that quarantined the failing tests has been merged, I think we should be good now.

**Comment by brybacki** [comment]
> /retest
> 
> The PR that quarantined the failing tests has been merged, I think we should be good now.

The test run in this PR are from code from the PR. To have the test quarantined one needs to rebase.

**Comment by xpivarc** [comment]
> > /retest
> > The PR that quarantined the failing tests has been merged, I think we should be good now.
> 
> The test run in this PR are from code from the PR. To have the test quarantined one needs to rebase.

Prow will rebase it before testing. We don't need to worry about it.

**Comment by fgimenez** [comment]
> > > /retest
> > > The PR that quarantined the failing tests has been merged, I think we should be good now.
> > 
> > 
> > The test run in this PR are from code from the PR. To have the test quarantined one needs to rebase.
> 
> Prow will rebase it before testing. We don't need to worry about it.

Yep, you can check here https://prow.ci.kubevirt.io/prowjob?prowjob=369f8f0c-c92c-11eb-904c-eecd3470eabc that the `base_sha` is https://github.com/kubevirt/kubevirt/commit/63f849f64428fd8ecdb033aa58ca6412a6fa6759

**Comment by brybacki** [comment]
Thanks for info. That is great, I forgot about that!
